### DISCUSSION TITLE
Mapping Toggle Event and Toggle value?

### DISCUSSION THREAD

**Discussion by JoergM-D** [nguyenquyhy/Flight-Tracker-StreamDeck#256] [Q&A]
Hi,
I'm looking for the right strings to control the ADF Soud toggle from the C172 (see screenshot)
![C172_panel](https://user-images.githubusercontent.com/85316002/121217216-eb5cdc80-c881-11eb-9f68-112f75df57e4.png)

NAV1, NAV2 and MKR Mute are working.  I tried as toggle event "radio_ADF_IDENT_TOGGLE" and as value ADF_Sound.
The toggle is working, the value not.
Is there anywhre a list, mapping, etc. which value works with which event?

Cheers Jörg

**Comment by rmroc451** [comment]
Can you give `ADF_IDENT` a try for the value?

**Comment by JoergM-D** [comment]
Thanks for your suggestion.
ADF_IDENT or ADF IDENT are not working. 
ADF SOUND and  ADF_CARD (no idea for what this is) also not work.
